src/tools/dev/scripts/visit-create-chksums

```diff
@@ -112,7 +112,8 @@ do
    $cmd INSTALL_NOTES.txt                                 >> $output
    $cmd jvisit$version.tar.gz                             >> $output
    $cmd VisIt-$version.dmg                                >> $output
-   $cmd visit$version2.darwin-x86_64.tar.gz               >> $output
+   $cmd visit$version2.darwin22-x86_64.tar.gz             >> $output
```

Comment:
I think we should change these to use file globs that do not use a *specific* version number of the associated OS (e.g. darwin 22 or debian 10 or fedora 31 or ubuntu 20). Otherwise, this file needs to be constantly updated as those version numbers change.